comtypes/test/test_excel.py

```diff
@@ -4,32 +4,43 @@
 import datetime
 import unittest
 
-import comtypes.test
-from comtypes.client import CreateObject
+from comtypes.client import CreateObject, GetModule
+
+################################################################
+#
+# TODO:
+#
+# It seems bad that only external test like this
+# can verify the behavior of `comtypes` implementation.
+# Find a different built-in win32 API to use.
+#
+################################################################
+
+try:
+    GetModule(("{00020813-0000-0000-C000-000000000046}",))  # Excel libUUID
+    from comtypes.gen.Excel import xlRangeValueDefault
+    IMPORT_FAILED = False
+except (ImportError, OSError):
+    IMPORT_FAILED = True
+
+
+@unittest.skipIf(IMPORT_FAILED, "This depends on Excel.")
+@unittest.skip("There is difference of `Range.Value` behavior "
+    "between Python >= 3.8.x and Python <= 3.7.x.")
+class BaseTest(unittest.TestCase):
```

Comment:
I would suggest to not inherit `BaseTest` from `unittest.TestCase` and do multiple inheritance in derived classes so they are inherited both from `BaseTest` and from `unittest.TestCase`. Declaring some class and then deleting it looks strange.

Another option is naming `test` method as non-test in base class and call it in derived class's test method which is a one liner or maybe an alias would work (didn't check it).